<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDetails</class>
 <widget class="QWidget" name="DeviceDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="idEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="messageContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>352</width>
               <height>202</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteDeviceButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdsButton">
              <property name="text">
               <string>Cmds</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cfgsButton">
              <property name="text">
               <string>Cfgs</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="cfgDetails"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="cmdDetails"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
